bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondDeviceHandler.java

```diff
@@ -0,0 +1,756 @@
+                        if (value == 0) {
+                            action = BondDeviceAction.TURN_FP_FAN_OFF;
+                            value = null;
+                        } else {
+                            action = BondDeviceAction.SET_FP_FAN;
+                        }
+                    } else {
+                        BondDeviceProperties devProperties = this.deviceProperties;
+                        if (devProperties != null) {
+                            int maxSpeed = devProperties.maxSpeed;
+                            value = (int) Math.ceil(((PercentType) command).intValue() * maxSpeed / 100);
+                        } else {
+                            value = 1;
+                        }
+                        if (value == 0) {
+                            action = BondDeviceAction.TURN_OFF;
+                            value = null;
+                        } else {
+                            action = BondDeviceAction.SET_SPEED;
+                        }
+                    }
+                    logger.trace("Fan speed command with speed set as {}", value);
+                    api.executeDeviceAction(deviceId, action, value);
+                } else if (command instanceof IncreaseDecreaseType) {
+                    logger.trace("Fan increase/decrease speed command");
+                    api.executeDeviceAction(deviceId,
+                            ((IncreaseDecreaseType) command == IncreaseDecreaseType.INCREASE
+                                    ? BondDeviceAction.INCREASE_SPEED
+                                    : BondDeviceAction.DECREASE_SPEED),
+                            null);
+                } else if (command instanceof OnOffType) {
+                    logger.trace("Fan speed command {}", command);
+                    if (devInfo.actions.contains(BondDeviceAction.TURN_FP_FAN_ON)) {
+                        action = command == OnOffType.ON ? BondDeviceAction.TURN_FP_FAN_ON
+                                : BondDeviceAction.TURN_FP_FAN_OFF;
+                    } else if (devInfo.actions.contains(BondDeviceAction.TURN_ON)) {
+                        action = command == OnOffType.ON ? BondDeviceAction.TURN_ON : BondDeviceAction.TURN_OFF;
+                    }
+                    if (action != null) {
+                        api.executeDeviceAction(deviceId, action, null);
+                    }
+                } else {
+                    logger.info("Unsupported command on fan speed channel");
+                }
+                break;
+
+            case CHANNEL_FAN_BREEZE_STATE:
+                logger.trace("Fan enable/disable breeze command");
+                api.executeDeviceAction(deviceId,
+                        command == OnOffType.ON ? BondDeviceAction.BREEZE_ON : BondDeviceAction.BREEZE_OFF, null);
+                break;
+
+            case CHANNEL_FAN_BREEZE_MEAN:
+                // TODO(SRGDamia1): write array command fxn
```

Comment:
Is this TODO comment still needed?

Reply:
Yes. I don't have a device that supports this, so would be unable to test it. Hopefully someone will pop up in the community asking for this feature, and I can coordinate with them to implement.